docs/software/load_balancer.md

```diff
@@ -1,12 +1,57 @@
 # Network Load Balancing
 
-## Overview
+## HAProxy - High performance TCP/HTTP load balancer
 
-!!! warning "Documentation in progress..." 
+HAProxy, which stands for High Availability Proxy, is a popular open source software TCP/HTTP Load Balancer and proxy solution. Its most common use is to improve the performance and reliability of a server environment by distributing the workload across multiple servers (e.g. web, application, database). 
 
-    The link redirects to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+It is best suited for high traffic web sites and powers quite a number of the world's most visited ones: GitHub, Imgur, Instagram, and Twitter. It has become the de-facto standard opensource load balancer, and it is often deployed by default in cloud platforms. 
 
-- [HAProxy - High performance TCP/HTTP load balancer.](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=30#p221)  
+!!! warning ""
+    This software title is recommended ONLY for advanced users !
 
-[Return to the **Optimised Software list**](../../dietpi_optimised_software)
+=== "Quick start"
+    After installation, you need to manually modify the `haproxy.cfg` to best fit your network requirements. Check the configuration manual [here](http://www.haproxy.org/#docs).
+
+    ``` bash
+    systemctl stop haproxy
+    nano /etc/haproxy/haproxy.cfg
+    systemctl start haproxy    
+    ```
+
+    Access web interface stats:
+    ``` bash
```

Comment:
Should there be an empty line above \`\`\`bash?

Reply:
I was wondering why Markdownlint did not complain. Probably it does not recognise this as fenced code then it is indented, or at least the rule is not applied then. I think otherwise it required empty lines around fenced code so lets keep it here as well.

Reply:
The option  
**MD031 - Fenced code blocks should be surrounded by blank lines**  
is not omitted via .markdownlintrc.

No idea.

Reply:
Empty line inserted